packages/app/src/search/SearchTabContainer.tsx

```diff
@@ -0,0 +1,50 @@
+import { useEffect, useState } from 'react';
+
+import SearchTab from './SearchTab';
+
+interface Props {
+  selectedPath: string;
+  onSelect: (path: string) => void;
+  searchValue: string;
+  setSearchValue: (value: string) => void;
+  getSearchablePaths: (path: string) => Promise<string[]>;
+}
+
+const BASE = '/';
+
+function SearchBoxContainer(props: Props) {
+  const {
+    selectedPath,
+    onSelect,
+    searchValue,
+    setSearchValue,
+    getSearchablePaths,
+  } = props;
+
+  const [paths, setPaths] = useState<string[]>();
+
+  useEffect(() => {
+    const fetchPaths = async () => {
+      setPaths(await getSearchablePaths(BASE));
+    };
+    if (!paths) {
+      void fetchPaths();
+    }
+  }, [paths, getSearchablePaths]);
```

Comment:
Since the fetch store was dropped in #1367, I use a simple `useEffect` to handle the fetching. This could be moved up in the hierarchy to cache the `paths`.

Reply:
Good for now. Eventually, it'd be good to show the search UI, let the user enter a search term, and show a spinner underneath the search field until the paths are fetched.